### PR TITLE
fix: vertically align close button for compact token

### LINKED ISSUE #1731: Token in compact mode is not vertically aligned
Screen Shot 2020-09-29 at 3 00 33 PM

The close icon is not vertically aligned in compact Tokens

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue
Closes #1731

## Description
Change close button padding to vertically center the icon inside.

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/token.scss [hunk 1]
```diff
@@ -109,7 +109,7 @@ $block: #{$fd-namespace}-token;
     line-height: 1.125rem;
 
     .#{$block}__close {
-      padding: $fd-token-compact-padding-vertical $fd-token-compact-padding-horizontal;
+      padding: ($fd-token-compact-padding-vertical * 2) $fd-token-compact-padding-horizontal $fd-token-compact-padding-vertical;
```
Comment (InnaAtanasova) [inline src/token.scss:112]: I'm not sure it's a good idea to change the paddings. You can easily fix the issue by changing the flex `alig-items` rule to normal:
Screen Shot 2021-04-28 at 12 19 22 PM

Comment (InnaAtanasova) [reply src/token.scss:112]: You will get this:
Screen Shot 2021-04-28 at 12 21 14 PM

Comment (N1XUS) [reply src/token.scss]: @InnaAtanasova done in a bit different way. Please re-check.

**Review by InnaAtanasova — APPROVED** [review]
@N1XUS you need a commit containing the text [ci visual] to trigger chromatic visual regression. Without it the checks won't pass.

**Pull request merged**